packages/js/src/plugins/typescript/plugin.ts

```diff
@@ -220,7 +261,7 @@ function buildTscTargets(
     };
   }
 
-  return targets;
+  return { targets, metadata: {} };
```

Comment:
We shouldn't have to return any metadata, that would avoid a lot of the snapshot changes too. I'm approving this, either way is fine but if you want to remove this it would make it a smaller changeset.